.github/workflows/pre-commit.yaml

```diff
@@ -6,6 +6,11 @@ on:
     branches: [main]
     paths-ignore:
       - "front/**"
+      - "node_modules/**"
+      - "yarn.lock"
+      - ".gitignore"
+      - "README.md"
```

Comment:
Actually I would keep the READE to avoid spaces at the end of the line, double spaces, etc...